tests/conftest.py

```diff
@@ -614,7 +619,9 @@ async def farmer_three_harvester_not_started(
 # because of a hack in shutting down the full node, which means you cannot run
 # more than one simulations per process.
 @pytest_asyncio.fixture(scope="function")
-async def daemon_simulation(bt, get_b_tools, get_b_tools_1):
+async def daemon_simulation(consensus_mode, bt, get_b_tools, get_b_tools_1):
+    if consensus_mode == Mode.HARD_FORK_2_0:
+        pytest.skip("Skipping test for Mode.HARD_FORK_2_0")
```

Comment:
I think it would make sense to explain that this test only supports one running at a time.
And the condition should probably be:
```
if consensus_mode != Mode.PLAIN:
```